AdblockPlusSafari/FilterList+Processing.m

```diff
@@ -0,0 +1,251 @@
+/*
+ * This file is part of Adblock Plus <https://adblockplus.org/>,
+ * Copyright (C) 2006-2015 Eyeo GmbH
+ *
+ * Adblock Plus is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * Adblock Plus is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/&gt.
+ */
+
+#import "FilterList+Processing.h"
+
+#import "AdblockPlus.h"
+
+#include <yajl/yajl_parse.h>
```

Comment:
We extract version and perform basic JSON validation using Yajl SAX parser. I do have experience with this parser so that I decided to use it over standard iOS approach.